aiohttp/web_ws.py

```diff
@@ -154,6 +154,13 @@ def send_bytes(self, data):
                             type(data))
         self._writer.send(data, binary=True)
 
+    def send_json(self, data, *, dumps=json.dumps):
+        if self._writer is None:
```

Comment:
These checks are not required: `.send_str()` performs them anyway.
There is no need to do the check twice.